### DISCUSSION TITLE
Spatial transcriptomics merge and SCTransform normalization

### DISCUSSION THREAD

**Discussion by Lisa-Salhi** [satijalab/seurat#4430] [Q&A]
Dear @satijalab Seurat team,

I am currently working with spatial transcriptomic data (10X Visium) from muscle biopsies. I have 5 healthy donors coming from 5 different patients (controls), and 5 biopsies of interest coming from 5 other patients. I chose to use the merge function and your SCTransform function. 
- Should I apply SCTransform before (on each sample), after the merge, or both? 
- Should I remove the genes with 0 UMI counts in all spots before performing the SCTransform normalization? 
- Finally, which variables should I use for the `vars.to.regress` parameter or how can I choose them?

Many thanks,

**Comment by jaisonj708** [comment]
We recommend applying SCT before merging since different datasets will have different models of technical noise, but it shouldn't make a huge difference if your data is all profiled by the same tech.
SCT will auto remove 0-count cells.
There is no necessity to specify vars.to.regress. But if you see unwanted sources of variation affecting your data (e.g, mitochondrial count percentage), you may add those.

**Reply by Lisa-Salhi** [reply]
Thanks a lot for your reply @jaisonj708 !

Ok, I'll apply SCT on each sample before merging. Should I apply it a second time, on the merged data?

The mean of reads per spot in each sample ranges from 39,000 to 133,000, and the data have not been been produced from the same slide (4 different slides). Would it make sense to apply SCT a second time on merged data, putting the slide (batch) as `vars.to.regress`?

**Comment by Lisa-Salhi** [comment]
Hello @jaisonj708 and @satijalab,

I finally applied two times SCT: one on each sample before merging, and one on merged data using `vars.to.regress="batch"` because some samples were on the same slide (4 slides total for 10 samples). For the second SCT, I also specified `assay="SCT"`. Is it correct to apply SCT on already SCT transformed data?

Best,
Lisa
